haiku-apps/catkeyseditor/catkeyseditor-0.1.1.recipe

```diff
@@ -10,7 +10,10 @@ HOMEPAGE="https://github.com/puckipedia/BeLocalized"
 COPYRIGHT="2014-2015 Puck Meerburg"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
OK, thanks @humdingerb for the info.

@korli, you did not approve the changes you requested in https://github.com/haikuports/haikuports/pull/921#pullrequestreview-12375620 (although it seems @dash102 has adressed all your comments in that review). Did you just forget to approve the changes or is there something else to change?
Thanks!